content/es/.spellcheck.yml

```diff
@@ -11,7 +11,7 @@
 matrix:
   - name: Spanish content
     sources:
-      - 'content/es/*.md'
+      - 'content/es/**/*.md'
```

Comment:
Is the rule `content/es/**/*.md` includes the rule `content/es/*.md`?
If not, then it might be good to preserve the rule `content/es/*.md`.

(In terms of consistency with L16-L17 of `.github/workflows/es-spellcheck.yml`)